doc/howto/write_an_alphabet/dna2_alphabet.cpp

```diff
@@ -91,12 +91,12 @@ struct dna2
 //! [writable_alphabet]
 
 //! [writable_alphabet_concept]
-static_assert(seqan3::Alphabet<dna2>);                   // ok
-static_assert(seqan3::WritableAlphabet<dna2>);           // ok
+static_assert(seqan3::alphabet<dna2>);                   // ok
+static_assert(seqan3::writable_alphabet<dna2>);           // ok
```

Comment:
Would be nice to preserve indentation, but it's quite a bit to change...

Reply:
Yes, but I don't think it's possible in a generic way.

Reply:
I mean, if you want to to go over the diff and correct for spacing with committable suggestions, I will of course integrate them. But it's a large diff.